### DISCUSSION TITLE
PHP8 Support

### DISCUSSION THREAD

**Discussion by wucdbm** [deployphp/deployer#2333] [General]
Heya, 

First of all, thank you for a great library!

Have you considered adding official support for PHP8 in the upcoming version 7?
Seems like a good idea to do it with a major version.

Anything I could help with, if that's the case?

Best!

**Comment by antonmedv** [comment]
Test beta11 it already supports  php8

**Comment by wucdbm** [comment]
Awesome! Thank you.
I'll give it a try next week!
